src/controllers/ArticleController.js

```diff
@@ -113,11 +113,17 @@ export default class ArticleController {
    * @returns {object} Object representing the response returned
    */
   static async update(req, res) {
```

Comment:
@djallas , After testing this method, I realized that it is not returning the updated article, it is just returning a message. I think it would be good to return the updated article in case we want to perform some action on it

Reply:
@rwajon check, it's updated